src/main/java/no/nav/familie/integrasjoner/kodeverk/domene/KodeverkDto.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.familie.integrasjoner.kodeverk.domene
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class KodeverkDto(@JsonProperty("betydninger") val betydninger: Map<String, List<BetydningDto>>)
+
+data class BetydningDto(@JsonProperty("gyldigFra") val gyldigFra: String,
+                        @JsonProperty("gyldigTil") val gyldigTil: String,
+                        @JsonProperty("beskrivelser") val beskrivelser: Map<String, BeskrivelseDto>)
+
+data class BeskrivelseDto(@JsonProperty("term") val term: String,
+                          @JsonProperty("tekst") val tekst: String)
+
+fun KodeverkDto.mapTerm() = betydninger.mapValues { it.value[0].beskrivelser[Språk.BOKMÅL.kode]?.term ?: "" }
```

Comment:
Må du ha en ? etter `it.value[0]` eller er det gitt at det alltid finnes et element i lista?

Reply:
Det blir OutOfIndexException hvis den ikke finnes. Mulig jag burde ta `firstOrNull()?.` her
Tanken var ett denne skulle brukes kun når man henter kodeverk uten historikk, då burde det kun finnes en. Men mulig man burde gjøre 2 Dto's, en som heter med historikk, så har man mapTerm kun på den uten historikk

Reply:
aha, ser den. Var ikke kjempelett å skjønne hva den egentlig gjør. Ikke kjenner jeg til api-et i noen grad heller.